packages/libs/react-ui/src/components/Button/Button.css.ts

```diff
@@ -37,11 +53,28 @@ export const container = style([
       backgroundColor: vars.colors.$neutral3,
       color: vars.colors.$neutral1,
       cursor: 'not-allowed',
+      pointerEvents: 'none',
     },
   },
 ]);
 
-const colors: Record<ColorType, ColorType> = {
+const rotate = keyframes({
+  '0%': { transform: 'rotate(0deg)' },
+  '100%': { transform: 'rotate(360deg)' },
+});
+
+export const buttonLoadingClass = style({
+  pointerEvents: 'none',
+});
```

Comment:
Is this necessary if it is being applied via the selector on the `container` class?

Reply:
I didn't realise I left the selector in. Will remove that as it doesn't work in VE.